12/umbraco-cms/content-delivery-api.md

```diff
@@ -395,15 +411,15 @@ Returns single or multiple items.
 {% endswagger-description %}
 
 {% swagger-parameter in="query" name="fetch" type="String" required="false" %}
-Structural query string option
+Structural query string option (e.g. "ancestors", "children", "descendants")
```

Comment:
Does it make sense to use inline code for these?

```suggestion
Structural query string option (e.g. `ancestors`, `children`, `descendants`)
```

Reply:
Yep, makes them more visible. Will fix 🙂